tests/test_schemas.py

```diff
@@ -21,12 +21,16 @@ def test_library_schema_full(self):
             ("/telemetry/proxy/api/v2/apmtelemetry", "$.payload.configuration[]"),
             ("/telemetry/proxy/api/v2/apmtelemetry", "$.payload"),  # APPSEC-52845
             ("/telemetry/proxy/api/v2/apmtelemetry", "$.payload.configuration[].value"),  # APMS-12697
-            ("/debugger/v1/input", "$[].dd.span_id"),  # DEBUG-2743
-            ("/debugger/v1/input", "$[].dd.trace_id"),  # DEBUG-2743
-            ("/debugger/v1/input", "$[].debugger.snapshot.probe.location.lines[]"),  # DEBUG-2743
-            ("/debugger/v1/input", "$[].debugger.snapshot.captures"),  # DEBUG-2743
         ]
 
+        if context.library == "python": # DEBUG-2743
```

Comment:
Can I check equality like this, or do I somehow need to take in account the library version?